src/core/core_structs/results.jl

```diff
@@ -84,3 +103,33 @@ function shorten_time_stamp(time::DataFrames.DataFrame)
     time = time[1:(size(time,1)-1),:]
     return time
 end
+
+function load_operation_results(folder_path::AbstractString, file_type)
+
+    if isfile(folder_path)
+        @error("not a folder path")
```

Comment:
This doesn't seem to be the correct type of error. cc. @daniel-thom

Reply:
Correct, that is an error log print only.  If you want to raise an exception then call `error()`